sites/fast-website/src/public/index.ejs

```diff
@@ -1,20 +1,21 @@
-<% const include = name => require(`./${name}.ejs`)(); %>
 <!DOCTYPE html>
 <html lang="en" dir="ltr" prefix="og: https://ogp.me/ns#">
     <head>
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title><%= htmlWebpackPlugin.options.title %></title>
-        <%= htmlWebpackPlugin.options.manifest %>
-        <meta property="twitter:title" content="<%= htmlWebpackPlugin.options.title %>" />
-        <meta name="title" property="og:title" content="<%= htmlWebpackPlugin.options.title %>" />
```

Comment:
The html-bundler-webpack-plugin provides the passed variables into a template under their names, without a long name space.